src/components/FormSelectSortBy/FormSelectSortBy.tsx

```diff
@@ -32,7 +32,7 @@ interface IFormSelectSortBy {
 const FormSelectSortBy: React.FC<IFormSelectSortBy> = React.memo(
   ({ sortItems, selectDispatchSortBy }) => {
     const classes = useStyles();
-    const sortBy: ISortBy = useSelector((state: AppStateType) => state.sortBy); //to do изменить any
+    const sortBy: ISortBy = useSelector((state: AppStateType) => state.sortBy);
```

Comment:
Move selectors to the separate file so that you can re-user them.
```
export { sortBySelector } from '<somewhere>/selectors';
...
const sortBy: ISortBy = useSelector(sortBySelector);
```